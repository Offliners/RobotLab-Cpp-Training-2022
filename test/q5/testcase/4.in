(10+14/18+(9+13/(11/3)-1+17-3+13))=
