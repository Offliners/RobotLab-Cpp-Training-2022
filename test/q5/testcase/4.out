3065
307
312
370
10
10
